optax/_src/transform_test.py

```diff
@@ -202,6 +202,35 @@ def test_centralize(self, inputs, outputs):
     centralized_inputs, _ = centralizer.update(inputs, None)
     chex.assert_tree_all_close(centralized_inputs, outputs)
 
+  @parameterized.parameters(
+    (0.01, 0.55, 123),
+    (0.3, 0.54, 234),
+    (1.0, 0.56, 314),
+  )
+  def test_add_noise(self, eta, gamma, seed):
+    # Prepare to compare noise with a rescaled unit-variance substitute.
+    noise = transform.add_noise(eta, gamma, seed)
+    noise_unit = transform.add_noise(1.0, 0.0, seed)
+
+    params = self.init_params
+    state = noise.init(params)
+    state_unit = noise_unit.init(params)
+
+    # Check the noise itself by adding it to zeros.
+    updates = jax.tree_map(jnp.zeros_like, self.per_step_updates)
```

Comment:
Nit: you could make this more self-contained by replacing `self.per_step_updates` with `params` directly (because that's ultimately the shape we want to match).

Reply:
Nice! Done.